tests/e2e/configurer/chain/commands.go

```diff
@@ -105,3 +114,24 @@ func (n *NodeConfig) CreateWallet(walletName string) string {
 	n.LogActionF("created wallet %s, waller address - %s", walletName, walletAddr)
 	return walletAddr
 }
+
+func (n *NodeConfig) QueryPropStatusTimed(wg *sync.WaitGroup, proposalNumber int, desiredStatus string, totalTime chan time.Duration) {
+	defer wg.Done()
```

Comment:
I think we can remove the waitgroup and instead send `1*time.Minute` on the `totalTime` channel since that's how long it takes at most for this query to wait. I think that would allow for the caller to get unblocked and continue. On the other hand, currently if `require.Eventually(...)` fails, the caller is blocked forever wiaitng on the result from `totalTime` channel that never arrives